fixing #634

alexcohn opened the pull request:
attempted to close file descriptor XX, expected to be unowned, actually owned by ParcelFileDescriptor XXX

tanersener commented:
Thanks Alex. I'll merge this but I don't plan to release it. I fixed this and other features I promised under [ffmpeg-kit](https://github.com/tanersener/ffmpeg-kit). As you know this project became too big. Even checking out master became an issue as reported in #516. Creating an empty project and applying your suggestions about downloading the source code of libraries when they're enabled was the only way to go forward. This is what I did in `ffmpeg-kit`. Also, to address all feature requests about macOS support I changed the scope and the name of the project. I'll move my `flutter` and `react-native` plugins under it too. I know this is a big change. And I don't expect everybody to be happy about it but I had to do this.

alexcohn commented:
I was busy with my daily job lately, so I was unaware of the change. I am
glad you are moving in this direction. Good luck with it!

Blessings,
Alex Cohn


On Sun, Mar 7, 2021 at 1:53 PM Taner Şener <notifications@github.com> wrote:

> Thanks Alex. I'll merge this but I don't plan to release it. I fixed this
> and other features I promised under ffmpeg-kit
> <https://github.com/tanersener/ffmpeg-kit>. As you know this project
> became too big. Even checking out master became an issue as reported in
> #516 <https://github.com/tanersener/mobile-ffmpeg/issues/516>. Creating
> an empty project and applying your suggestions about downloading the source
> code of libraries when they're enabled was the only way to go forward. This
> is what I did in ffmpeg-kit. Also, to address all feature requests about
> macOS support I changed the scope and the name of the project. I'll move my
> flutter and react-native plugins under it too. I know this is a big
> change. And I don't expect everybody to be happy about it but I had to do
> this.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/tanersener/mobile-ffmpeg/pull/690#issuecomment-792265037>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAFTZM4OIJVN4GSFFYZOBFTTCNSMZANCNFSM4YXDC25Q>
> .
>
